### DISCUSSION TITLE
Next git-lfs release?

### DISCUSSION THREAD

**Discussion by lambiase** [git-lfs/git-lfs#4638] [Q&A] [answered]
Hi,

I was wondering if you plan a new git-lfs release any time soon.
In my case, the PR https://github.com/git-lfs/git-lfs/pull/4418 from @bk2204 (already on `main`) fixed issues I was having with network shares on Windows.
Thanks!

**Comment by bk2204** [comment, marked as answer]
Hey,

The release is scheduled for tomorrow, September 24.

**Reply by lambiase** [reply]
Thanks!
